roles/openshift_health_checker/openshift_checks/memory_availability.py

```diff
@@ -0,0 +1,31 @@
+# pylint: disable=missing-docstring
+from openshift_checks import OpenShiftCheck, get_var
+from openshift_checks.mixins import NotContainerizedMixin
+
+
+class MemoryAvailability(OpenShiftCheck):
+    """Check that recommended memory is available."""
+
+    name = "memory_availability"
+    tags = ["preflight"]
+
+    recommended_node_memory_gb = 8.0
+    recommended_master_memory_gb = 16.0
+
+    def run(self, tmp, task_vars):
+        group_names = get_var(task_vars, "group_names", default=[])
+        memoryfree = get_var(task_vars, "facter_memoryfree")
+
+        recommended_memory_gb = self.recommended_master_memory_gb
```

Comment:
Masters *should* be nodes, but in general they are unschedulable nodes, so they do not require additional disk space.